assets/javascripts/discourse/initializers/extend-for-translate-button.js.es6

```diff
@@ -1,5 +1,3 @@
-import PostMenuComponent from 'discourse/components/post-menu';
```

Comment:
This is a duplicate of https://github.com/discourse/discourse-solved/commit/ed0a2948cbb4b63d0af0f365b24296044c762e49 essentially. discourse@master removed the `post-menu` source file so the import fails on file load.